install_config/monitoring/configuring-alertmanager.adoc

```diff
@@ -0,0 +1,112 @@
+[[configuring-alertmanager]]
+== Configuring Alertmanager
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+The Alertmanager manages incoming alerts, including silencing, inhibition, aggregation and sending out notifications through methods such as email, PagerDuty and HipChat.
+
+The default configuration of the OpenShift Monitoring Alertmanager cluster is:
+
+----
+  global:
+    resolve_timeout: 5m
+  route:
+    group_wait: 30s
+    group_interval: 5m
+    repeat_interval: 12h
+    receiver: default
+    routes:
+    - match:
+        alertname: DeadMansSwitch
+      repeat_interval: 5m
+      receiver: deadmansswitch
+  receivers:
+  - name: default
+  - name: deadmansswitch
+----
+
+This configuration can be overwritten using the Ansible variable `openshift_cluster_monitoring_operator_alertmanager_config` from the `openshift_cluster_monitoring_operator` role.
+
+The following example configures link:https://www.pagerduty.com/[PagerDuty] for notifications. See the PagerDuty documentation for link:https://www.pagerduty.com/docs/guides/prometheus-integration-guide/[Alertmanager] to learn how to retrieve the `service_key`.
```

Comment:
It is better to open one new  page when clicking the [PagerDuty] and [Alertmanager] link